CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Contributing Instructions
+
+### Setup
+
+```sh
+# install golang, then:
+go get -u github.com/goware/modvendor
+make vendor
+```
+
+### Compiling and running tests
+
+```sh
+make build
+make test
+# see the others in the Makefile
```

Comment:
Maybe we can directly reference the integration tests here, like this:

---

### Compiling and running tests

To compile the collector and helper binaries:

```
make build
```

After building the collector you can find the binary in the repository folder:

```
./pganalyze-collector --help
```

To run the unit tests:

```
make test
```

To run the integration tests:

```
make integration_test
```

Note the integration tests require Docker, and will take a while to run through.